src/float.rs

```diff
@@ -2247,6 +2247,7 @@ macro_rules! float_const_impl {
         #[allow(non_snake_case)]
         pub trait FloatConst {
             $(#[$doc] fn $constant() -> Self;)+
+            #[doc = "Return the full circle constant `τ`."] fn TAU() -> Self;
```

Comment:
It's a breaking change to add items to a trait without a default implementation, and there are no super-traits on `FloatConst` to give us the means to compute a default. However, if you add a condition `where Self: Add<Self, Output = Self>` on this method, we can add a default `Self::PI() + Self::PI()`.

Reply:
Sure. Done.